packages/http-client-python/emitter/src/lib.ts

```diff
@@ -17,9 +17,7 @@ export interface PythonEmitterOptions {
   debug?: boolean;
   flavor?: "azure";
   "examples-dir"?: string;
-  // If true, package namespace will respect the typespec namespace. Otherwise,
-  // package namespace is always aligned with package name.
-  "enable-typespec-namespace"?: boolean;
+  namespace?: string;
```

Comment:
i'll get my pr merged first so you won't need to add a `namespace` config since tcgc will handle it